vignettes/bds_exposure.Rmd

```diff
@@ -361,7 +361,7 @@ For example, to calculate the total dose by subject and treatment,
 adex <- derive_param_exposure(
   adex,
   dataset_add = adex,
-  by_vars = exprs(STUDYID, USUBJID, !!!adsl_vars),
+  by_vars = c(get_admiral_option("subject_keys"), adsl_vars),
```

Comment:
Either we should update all calls in the vignette or none.

@bms63 , did we reach a decision regarding updating vignettes and templates?

Reply:
I went ahead an replaced all the incidents in this vignette until we decide to replace all the others.